.orchestra/ci/install-system-dependencies.sh

```diff
@@ -90,6 +90,7 @@ PACKAGES+=(p7zip-full)
 PACKAGES+=(winbind)
 PACKAGES+=(wine)
 PACKAGES+=(wine32)
+PACKAGES+=(wine64)
```

Comment:
Squash in the commit introducing support for 64-bit stuff

Reply:
I kept this separate because I'm hesitant as to why it wasn't included already, so I wanted second opinion on this.

But from you comment, I guess this makes sense, so squashing it is.

Reply:
I put together the code in a hurry to unlock you. I didn't expect it to be perfect.